schema/src/lib.rs

```diff
@@ -47,7 +47,7 @@ pub trait IntoSchema {
 /// Applicable for types that represents decimal place of fixed point
 pub trait DecimalPlacesAware {
     /// decimal places of fixed point
-    fn decimal_places() -> usize;
+    fn decimal_places() -> u64;
```

Comment:
This can be safely assumed to be within u32. The fixed point number used here is a single i64, it can't possibly have more decimal places than u32. (It can have more decimal places than the mantissa of i64, but that fits into a single byte).